src/status_im2/contexts/chat/lightbox/constants.cljs

```diff
@@ -16,4 +16,11 @@
 
 (def ^:const drag-threshold 100)
 
-(def ^:const image-description-in-lightbox? false)
+;;; TEXT SHEET
+(def ^:const text-min-height 68)
+
+(def ^:const text-margin 12)
+
+(def ^:const bar-container-height 20)
+
+(def ^:const line-height 22)
```

Comment:
Just my style preference, but I think a long list of constants can be more tightly packed, like so:

```clojure
(ns status-im2.contexts.chat.lightbox.constants)

(def ^:const small-image-size 40)
(def ^:const focused-extra-size 16)
(def ^:const focused-image-size (+ small-image-size focused-extra-size))
(def ^:const small-list-height 80)
(def ^:const small-list-padding-vertical 12)
(def ^:const top-view-height 56)
(def ^:const separator-width 16)
(def ^:const drag-threshold 100)

;;; TEXT SHEET

(def ^:const text-min-height 68)
(def ^:const text-margin 12)
(def ^:const bar-container-height 20)
(def ^:const line-height 22)
```